src/components/modals/AddDacModal.js

```diff
@@ -1,27 +1,42 @@
+import _ from 'lodash';
 import { Component } from 'react';
 import { div, form, h, hh, input, label, textarea } from 'react-hyperscript-helpers';
-import AsyncSelect from 'react-select/lib/Async';
+import AsyncSelect from 'react-select/async';
+import { DAC } from '../../libs/ajax';
 import { Alert } from '../Alert';
 import { BaseModal } from '../BaseModal';
-import { DAC } from '../../libs/ajax';
+import { DacUsers } from '../DacUsers';
+
 
 export const AddDacModal = hh(class AddDacModal extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
       isEditMode: this.props.isEditMode,
-      dac: this.props.isEditMode ? this.props.dac : {},
+      dac: this.props.isEditMode ? this.props.dac : { dac: {}, chairpersons: [], members: [] },
       error: {
         show: false,
         title: '',
         msg: ['']
-      }
+      },
+      chairsToAdd: [],
+      chairsSelectedOptions: [],
+      chairIdsToRemove: [],
+      membersToAdd: [],
+      membersSelectedOptions: [],
+      memberIdsToRemove: [],
     };
 
     this.closeHandler = this.closeHandler.bind(this);
     this.afterOpenHandler = this.afterOpenHandler.bind(this);
     this.OKHandler = this.OKHandler.bind(this);
+    this.userSearch = this.userSearch.bind(this);
+    this.onChairSearchChange = this.onChairSearchChange.bind(this);
+    this.onMemberSearchChange = this.onMemberSearchChange.bind(this);
+    this.removeChairperson = this.removeChairperson.bind(this);
+    this.removeMember = this.removeMember.bind(this);
+    this.handleChange = this.handleChange.bind(this);
   }
 
   componentDidMount() {
```

Comment:
not used, should be removed ?

Reply:
Still very much a work in progress, not ready for review.